README.md

```diff
@@ -87,3 +87,20 @@ vim.g.asyncrun_program = vim.empty_dict()
 vim.api.nvim_command("let g:asyncrun_program.vcvars64 = { opts -> '\"C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Auxiliary/Build/vcvars64.bat\" && ' .. opts.cmd }")
 vim.g.cmake_asyncrun_options = vim.tbl_extend('force', vim.g.cmake_asyncrun_options, {program = 'vcvars64'})
 ```
+
+## DAP configuration example
+
+You can customize the DAP configuration via `g:cmake_dap_configuration`:
+
+```lua
+vim.g.cmake_dap_configuration = {
+  type = "codelldb",
```

Comment:
Can I ask you to use single quotes? I just use single quotes in the whole codebase :)

Reply:
Sorry, I should use single quotes. I saw the `stylua.toml` file but forgot the code in markdown would not auto-formatted...